create-mern-ts-app.js

```diff
@@ -45,6 +46,13 @@ function createProjectTemplate(projectName) {
   fs.copySync(srcRoot, destRoot);
 }
 
+function generateEnvFile(projectName) {
+  const destRoot = path.resolve(projectName);
+  const envContents = fs.readFileSync(path.join(destRoot, 'backend', '.env.example'), 'utf8')
+    .replace(/auth-shared-secret/g, crypto.randomBytes(24).toString('hex'));
+  fs.writeFileSync(path.join(destRoot, 'backend', '.env'), `# This is an auto-generated file, change at your own will and risk.\n\n${envContents}`);
```

Comment:
Cool, thanks for the changes!
I guess this only adds a new line instead of replacing the previous content?
Do you feel you could remove the previous comments in this PR or do you think we should open a new issue for that? Your call!

Reply:
I guess it would make more sense to do it here. Assuming it works :p I probably won't have time to do it today, so you'll probably have to wait for tomorrow :)

Reply:
Ok no hurry! I'd be glad if you could solve it! Thanks again! :)

Reply:
There's quite a lot with work atm, so I'll have to put this on the shelf for now. Sorry! I'll try and fix it ASAP

Reply:
Absolutely! If you've got the time, please do :)